### ISSUE TITLE
Terraform 0.13 support

### ISSUE THREAD

**Issue by dirk39** [kjagiello/terraform-aws-codepipeline-slack-notifications#1]
Hi @kjagiello 
Thanks for your great job! I really appreciate it.

Have you considered to support Terraform 0.13? 

Thanks 

Dirk

**Comment by kjagiello** [comment]
Sure thing! Just being curious, but which version of the AWS provider are you using? This module is currently only tested with 2.X, but not with 3.X. I could give it a spin on 3.X and see if its compatible, then update `versions.tf` accordingly.

**Comment by dirk39** [comment]
I'm actually using the version 3.1 of the AWS provider

**Comment by kjagiello** [comment]
Alright! I'll try to test the module this weekend and see if it's 3.X-compatible. It probably is, but better safe than sorry!

**Comment by dirk39** [comment]
Another thing I was thinking is to use codestar-notification instead of creating a custom Cloudwatch event. If ok for you I can work on it.

**Comment by kjagiello** [comment]
I'm not familiar with AWS CodeStar, but will take a look at get back to you 🙌

**Comment by kjagiello** [comment]
Took a quick look and it looks promising. Seems to be much less effort to manage CodePipeline using that, instead of the Cloudwatch events. It would be great if you could work on that!

**Comment by dirk39** [comment]
Great! I will open a PR as soon as I can :+1:

**Comment by jchadwick-smug** [comment]
Are there plans to make a new release (with TF 0.13 support, per issues/2) anytime soon?

**Comment by jchadwick-smug** [comment]
Attempting to use `git::https://github.com/kjagiello/terraform-aws-codepipeline-slack-notifications.git?ref=terraform-0.13` as a `source` results in this problem:

```
Initializing modules...
Downloading git::https://github.com/kjagiello/terraform-aws-codepipeline-slack-notifications.git?ref=terraform-0.13 for codepipeline_notifications...
- codepipeline_notifications in .terraform/modules/codepipeline_notifications
Downloading git::https://github.com/cloudposse/terraform-terraform-label.git?ref=tags/0.4.0 for codepipeline_notifications.default_label...
- codepipeline_notifications.default_label in .terraform/modules/codepipeline_notifications.default_label

Error: Unsupported Terraform Core version

  on .terraform/modules/codepipeline_notifications.default_label/versions.tf line 2, in terraform:
   2:   required_version = "~> 0.12.0"
```

Looks like https://github.com/kjagiello/terraform-aws-codepipeline-slack-notifications/blob/terraform-0.13/main.tf#L2 needs to be bumped to version `0.5.0`, where TF 0.13 support was added: https://github.com/cloudposse/terraform-terraform-label/releases

**Comment by kjagiello** [comment]
Just released [v1.1.0](https://github.com/kjagiello/terraform-aws-codepipeline-slack-notifications/releases/tag/v1.1.0) that features support for TF 0.12+. Thanks for all your contributions and reports and let me know if you run into any issues!
